--- /mnt/live/memory/bundles/04-kde.sb/root/.kde/share/config/plasma-desktop-appletsrc	2013-03-16 09:34:08.000000000 +0000
+++ /root/.kde/share/config/plasma-desktop-appletsrc	2013-04-02 10:35:36.554042017 +0000
@@ -44,6 +44,15 @@
 [Containments][1][Applets][17][LayoutInformation]
 Order=1
 
+[Containments][1][Applets][19]
+geometry=52,4,48,48
+immutability=1
+plugin=icon
+zvalue=5
+
+[Containments][1][Applets][19][Configuration]
+Url=file:///usr/share/applications/kde4/dolphin.desktop
+
 [Containments][1][Applets][2]
 geometry=0,4,48,48
 immutability=1
@@ -166,7 +175,7 @@
 ActionPluginsSource=Global
 activity=New Activity
 activityId=a0b281ef-4afd-4b11-9eea-1575eec99f40
-alignToGrid=false
+alignToGrid=true
 blankLabel=false
 clickForFolderPreviews=true
 customIconSize=48
@@ -186,9 +195,10 @@
 mimeFilter=
 numTextLines=2
 plugin=folderview
+savedPositions=1,4,blockchain.desktop,10,220,brainwallet.desktop,10,325,electrum.desktop,10,115,Home.desktop,10,10
 screen=0
 showPreviews=true
-sortColumn=0
+sortColumn=-1
 url=desktop:/
 wallpaperplugin=image
 wallpaperpluginmode=SingleImage
